foodji_ui/lib/l10n/app_en.arb

```diff
@@ -63,6 +67,15 @@
   "category_bread": "Bread",
   "category_snack": "Snack",
 
+  "form_is_required": "is Required",
```

Comment:
Ça se peut que je me trompe vu l'absence de contexte, mais ça devrait pas être "Is required"? En gros, le premier caractère seulement est en majuscule?

Reply:
Je l'utilise pour compléter les phrases de validation d'une recette.. ça vaudrait la peine que je mette un commentaire à côté, mais la syntaxe est bonne